src/civetweb.c

```diff
@@ -5856,6 +5861,9 @@ static void handle_static_file_request(struct mg_connection *conn,
 	                "Content-Type: %.*s\r\n"
 	                "Content-Length: %" INT64_FMT "\r\n"
 	                "Connection: %s\r\n"
+	               	"Cache-Control: no-cache, no-store, must-revalidate\r\n"
+			"Pragma: no-cache\r\n"
+			"Expires: 0\r\n"
```

Comment:
I've met an issue with this. In my case some "static files" change in time, but because of the cache i don't get the last version. I have to browse in incognito mode to bypass the issue.